keyboards/wookong/wk87/info.json

```
@@ -0,0 +1,268 @@
+            {"matrix": [3, 4], "x": 114, "y": 40, "flags": 4},
+            {"matrix": [3, 3], "x": 127, "y": 40, "flags": 4},
+            {"matrix": [3, 2], "x": 140, "y": 40, "flags": 4},
+            {"matrix": [3, 1], "x": 153, "y": 40, "flags": 4},
+            {"matrix": [3, 0], "x": 174, "y": 40, "flags": 1},
+            {"matrix": [4, 0], "x": 8, "y": 52, "flags": 1},
+            {"matrix": [4, 1], "x": 29, "y": 52, "flags": 4},
+            {"matrix": [4, 2], "x": 42, "y": 52, "flags": 4},
+            {"matrix": [4, 3], "x": 55, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 68, "y": 52, "flags": 4},
+            {"matrix": [4, 5], "x": 81, "y": 52, "flags": 4},
+            {"matrix": [4, 6], "x": 94, "y": 52, "flags": 4},
+            {"matrix": [4, 7], "x": 107, "y": 52, "flags": 4},
+            {"matrix": [4, 8], "x": 120, "y": 52, "flags": 4},
+            {"matrix": [4, 9], "x": 133, "y": 52, "flags": 4},
+            {"matrix": [4, 10], "x": 146, "y": 52, "flags": 4},
+            {"matrix": [4, 11], "x": 171, "y": 52, "flags": 1},
+            {"matrix": [4, 15], "x": 211, "y": 52, "flags": 1},
+            {"matrix": [5, 16], "x": 2, "y": 64, "flags": 1},
+            {"matrix": [5, 15], "x": 18, "y": 64, "flags": 1},
+            {"matrix": [5, 14], "x": 34, "y": 64, "flags": 1},
+            {"matrix": [5, 11], "x": 83, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 131, "y": 64, "flags": 1},
+            {"matrix": [5, 9], "x": 148, "y": 64, "flags": 1},
+            {"matrix": [5, 8], "x": 164, "y": 64, "flags": 1},
+            {"matrix": [5, 5], "x": 180, "y": 64, "flags": 1},
+            {"matrix": [5, 2], "x": 198, "y": 64, "flags": 1},
+            {"matrix": [5, 1], "x": 211, "y": 64, "flags": 1},
+            {"matrix": [5, 0], "x": 224, "y": 64, "flags": 1}
+        ]
+    },
+    "url": "https://github.com/hypharf",
+    "usb": {
+        "device_version": "1.0.0",
+        "force_nkro": true,
+        "pid": "0x6322",
+        "vid": "0x414B"
+    },
+    "ws2812": {
+        "pin": "B15"
+    },
+    "layouts": {
+       "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [0, 3], "x": 3, "y": 0},
+                {"matrix": [0, 4], "x": 4, "y": 0},
+                {"matrix": [0, 5], "x": 5, "y": 0},
+                {"matrix": [0, 6], "x": 6.5, "y": 0},
+                {"matrix": [0, 7], "x": 7.5, "y": 0},
+                {"matrix": [0, 8], "x": 8.5, "y": 0},
+                {"matrix": [0, 9], "x": 9.5, "y": 0},
+                {"matrix": [0, 10], "x": 11, "y": 0},
+                {"matrix": [0, 11], "x": 12, "y": 0},
+                {"matrix": [0, 12], "x": 13, "y": 0},
+                {"matrix": [0, 13], "x": 14, "y": 0},
+                {"matrix": [0, 15], "x": 15.6, "y": 0},
+                {"matrix": [0, 16], "x": 16.9, "y": 0},
```

denote keys are, also, encoders
```suggestion
                {"matrix": [0, 15], "x": 15.6, "y": 0, "encoder": 0},
                {"matrix": [0, 16], "x": 16.9, "y": 0, "encoder": 1},
```

---

Done